src/imagej/images.py

```diff
@@ -1,6 +1,47 @@
 """
 Utility functions for creating and working with images.
 """
+import logging
+
+import numpy as np
+import scyjava as sj
+
+from imagej._java import jc
+
+_logger = logging.getLogger(__name__)
+
+
+# fmt: off
+_imglib2_types = {
+    # "net.imglib2.type.logic.BitType":                               "bool",
+    "net.imglib2.type.numeric.integer.ByteType":                      "int8",
+    "net.imglib2.type.numeric.integer.ByteLongAccessType":            "int8",
+    "net.imglib2.type.numeric.integer.ShortType":                     "int16",
+    "net.imglib2.type.numeric.integer.ShortLongAccessType":           "int16",
+    "net.imglib2.type.numeric.integer.IntType":                       "int32",
+    "net.imglib2.type.numeric.integer.IntLongAccessType":             "int32",
+    "net.imglib2.type.numeric.integer.LongType":                      "int64",
+    "net.imglib2.type.numeric.integer.LongLongAccessType":            "int64",
+    "net.imglib2.type.numeric.integer.UnsignedByteType":              "uint8",
+    "net.imglib2.type.numeric.integer.UnsignedByteLongAccessType":    "uint8",
+    "net.imglib2.type.numeric.integer.UnsignedShortType":             "uint16",
+    "net.imglib2.type.numeric.integer.UnsignedShortLongAccessType":   "uint16",
+    "net.imglib2.type.numeric.integer.UnsignedIntType":               "uint32",
+    "net.imglib2.type.numeric.integer.UnsignedIntLongAccessType":     "uint32",
+    "net.imglib2.type.numeric.integer.UnsignedLongType":              "uint64",
+    "net.imglib2.type.numeric.integer.UnsignedLongLongAccessType":    "uint64",
+    # "net.imglib2.type.numeric.ARGBType":                            "argb",
+    # "net.imglib2.type.numeric.ARGBLongAccessType":                  "argb",
+    "net.imglib2.type.numeric.real.FloatType":                        "float32",
+    "net.imglib2.type.numeric.real.FloatLongAccessType":              "float32",
+    "net.imglib2.type.numeric.real.DoubleType":                       "float64",
+    "net.imglib2.type.numeric.real.DoubleLongAccessType":             "float64",
+    # "net.imglib2.type.numeric.complex.ComplexFloatType":            "cfloat32",
+    # "net.imglib2.type.numeric.complex.ComplexFloatLongAccessType":  "cfloat32",
+    # "net.imglib2.type.numeric.complex.ComplexDoubleType":           "cfloat64",
+    # "net.imglib2.type.numeric.complex.ComplexDoubleLongAccessType": "cfloat64",
```

Comment:
Is there a reason we are migrating the commented-out mappings as well?

Reply:
They are useful in case we ever want to work on enabling them in the future, since they identify what the mapping *would* be, with additional effort. It's a starting hint. Otherwise, that research/knowledge is lost and must be redone.

Reply:
Alright, I don't feel strongly. Let's just leave it.